code/modules/economy/price_list.dm

```diff
@@ -1011,16 +1016,54 @@
 /obj/item/mech_component/price_tag = 150
 /obj/item/mech_equipment/price_tag = 200
 
-/obj/item/weapon/gun/launcher/grenade/price_tag = 1500
-
-/obj/item/weapon/storage/briefcase/price_tag = 50
-
 /obj/item/inflatable/price_tag = 40
 
 /obj/item/weapon/tool/knife/dagger/bluespace/price_tag = 400
 
-
 /obj/item/weapon/reagent_containers/food/snacks/meat/roachmeat/price_tag = 75
 /obj/item/weapon/reagent_containers/food/snacks/meat/roachmeat/seuche/price_tag = 100
 /obj/item/weapon/reagent_containers/food/snacks/meat/roachmeat/kraftwerk/price_tag = 100
 /obj/item/weapon/reagent_containers/food/snacks/meat/roachmeat/jager/price_tag = 125
+
+/obj/item/toy/price_tag = 40
+/obj/item/device/toner/price_tag = 50
+
+/obj/item/device/camera_film/price_tag = 25
+
+/obj/item/device/camera/price_tag = 50
+/obj/item/weapon/storage/photo_album/price_tag = 50
+
+/obj/item/weapon/wrapping_paper/price_tag = 20
+/obj/item/weapon/packageWrap/price_tag = 20
+/obj/item/weapon/mop/price_tag = 15
+/obj/item/weapon/caution/price_tag = 10
+/obj/item/weapon/storage/bag/trash/price_tag = 25
+
+/obj/item/weapon/storage/lunchbox/price_tag = 25
+
+/obj/item/weapon/storage/briefcase/price_tag = 50
+/obj/item/weapon/soap/nanotrasen/price_tag = 60
+/obj/item/weapon/storage/pouch/price_tag = 100
+/obj/item/weapon/storage/pouch/ammo/price_tag = 200
+/obj/item/weapon/storage/pouch/tubular/price_tag = 140
+/obj/item/weapon/storage/pouch/medium_generic/price_tag = 255
+/obj/item/weapon/storage/pouch/large_generic/price_tag = 410
+
+/obj/item/weapon/rig/price_tag = 300
```

Comment:
![unknown](https://user-images.githubusercontent.com/61743710/125924403-6da96ffc-ef8e-4548-86cf-5d022599bb82.png)

Reply:
Based.

Jokes on you, the actual price of rigs rounds the 1k. Because it counts all their components, and modules. Including how much filled is the O2 Tank.